### ISSUE TITLE
[ui5-button]: Cannot Be Activated Using Enter Key In VPC Mode

### ISSUE THREAD

**Issue by ee92** [SAP/ui5-webcomponents#11703]
### Bug Description

When using JAWS with VPC mode, the <ui5-button> cannot be activated by keyboard

### Affected Component

Button

### Expected Behaviour

Expect to be able to trigger the button action using keyboard "Enter" key when using VPC mode

### Isolated Example

https://sap.github.io/ui5-webcomponents/components/main/Button

### Steps to Reproduce

1. go to Button playground
2. add `accessible-role="Link" onclick="window.open('https://sap.com','_blank');"` to button html
3. tab to the button
4. notice "Enter" key DOES tigger navigation
5. turn on JAWS with VPC mode and focus the button
6. notice "Enter" key does not trigger the navigation

### Priority

Low

### UI5 Web Components Version

2.10.0

### Browser

Chrome

### Organization

SuccessFactors

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by Valkanister** [comment]
Hello @SAP/ui5-webcomponents-topic-b , t

Please have a look why the "Enter" event is not triggered when VPC mode of the screen reader is ON. 
I tested with JAWS in the Playground as described. 

Kind regards, 

Georgi

SAP Development Support

**Comment by didip1000** [comment]
Hi @ee92,

Thanks for reporting this. This behavior is a known issue with the JAWS screen reader, and it's not specific to our components. You can see that the same is reproducible with a regular HTML button here: https://jsfiddle.net/unazko/0ngyh7fL/3/

The issue has already been reported to the JAWS development team, which you can follow for progress updates here: https://github.com/FreedomScientific/standards-support/issues/899 

Note that the issue is reported for the `space` key but as described in the issue, it's caused by the fact that the “keydown” event is not fired at all when JAWS with VPC is started.

Don't hesitate to re-open the ticket if you have any further questions.

Best Regards,
Diana
